src/main/kotlin/ConfigTypes.kt

```diff
@@ -55,9 +56,12 @@ data class VersionConfig(val version: String)
  * [statusMessage] is the bot status message on Discord
  * [statusMessageType] is the type of status. Playing is 0, Streaming is 1, Listening is 2 and Watching is 3.
  * [defaultGithubUser] is the default user / org used in the [IssueCommand]
+ * [downloadChannel] is the channel id for download counter.
+ * [secondaryDownloadChannel] is the channel id for the secondary download counter.
```

Comment:
Are these compatible with regular channels and not just voice? If so, make it clear you can use voice, eg "is the ID for the voice / text channel"